### ISSUE TITLE
how do i fix this problem when i use psnr

### ISSUE THREAD

**Issue by s0966066980** [Wenchao-Du/LIR-for-Unsupervised-IR#11]
![image](https://user-images.githubusercontent.com/45198905/127871569-d876bc71-7160-4c0b-b7e6-85b7b634dddc.png)

**Comment by s0966066980** [comment]
![image](https://user-images.githubusercontent.com/45198905/127873688-5264d51d-b018-416e-93ab-9d03481c02e7.png)

**Comment by Wenchao-Du** [comment]
The first problem should use the output.detach() instead of outputs directly. For the second problem,  you should check if the opts.input is the image path ? good luck!

**Comment by s0966066980** [comment]
Thanks for replying, I still have this problem, or does the image have a format requirement? image is .jpg
